test/can-view-parser-test.js

```diff
@@ -852,7 +838,9 @@ testHelpers.dev.devOnlyTest('Fix false warning on missing closed quote (canjs/ca
 		equal(count, teardown());
 	};
 
+	/* jshint ignore:start */
```

Comment:
why is this needed?

Reply:
Because jshint complains from unused variable

Reply:
Why is the variable needed I mean.

Reply:
Is used in stache template